### PR TITLE
Support getSubscribers & getsubscription for nknc pubsub

### PR DESCRIPTION
**Pull request by gdmmx**
Signed-off-by: gdmmx <gdmmx@163.com>

### Proposed changes in this pull request
Support getSubscribers & getsubscription for nknc pubsub

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [x] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [ ] Documentation Improvement

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [x] Read the [CONTRIBUTION guidelines](https://github.com/nknorg/nkn#contributing).
- [x] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [ ] Added proper documentation where ever applicable (in code and README.md).
- [x] Code has been written according to [NKN-Golang-Style-Guide](https://github.com/nknorg/nkn/wiki/NKN-Golang-Style-Guide)

### Extra information
Any extra information related to this pull request.

### CODE REVIEW

File: cli/pubsub/pubsub.go [hunk 1]
```diff
@@ -81,6 +81,23 @@ func subscribeAction(c *cli.Context) error {
...
 		buff, _ := txn.Marshal()
 		resp, err = client.Call(Address(), "sendrawtransaction", 0, map[string]interface{}{"tx": hex.EncodeToString(buff)})
+	case c.Bool("get"):
+		topic := c.String("topic")
+		if topic == "" {
+			fmt.Println("topic is required with [--topic]")
+			return nil
+		}
+
+		params := map[string]interface{}{"topic": topic}
```
Comment (yilunzhang) [inline cli/pubsub/pubsub.go]: There are a few additional parameters:
* offset (default 0)
* limit (default 1000)
* txPool (default false)
* meta (default false)

We can consider adding them as optional arguments

Comment (gdmmx) [reply cli/pubsub/pubsub.go]: Setting offset & limit was supported by this commit. The default value will be 0 & 1000 at node side if no setting by cli.

It is not friendly for appointed meta(json format) via cli option. Query with txPool or meta is a small crowd requirements for a cli tools.
Let's defer it until met actual demand.

File: cli/pubsub/pubsub.go [hunk 2]
```diff
@@ -81,6 +81,23 @@ func subscribeAction(c *cli.Context) error {
...
+		subscriber := c.String("subscriber")
+		if subscriber == "" { // get list of topic
+			params["offset"] = c.Uint64("offset")
+			params["limit"] = c.Uint64("count")
```
Comment (yilunzhang) [inline cli/pubsub/pubsub.go]: Suggest to rename `count` to `limit` for API consistency and naming convention

Comment (gdmmx) [reply cli/pubsub/pubsub.go]: Done.

**Pull request merged**